.github/workflows/sui-protocol-version-monitor.yml

```diff
@@ -15,7 +15,7 @@ on:
         description: 'Environment'
         required: true
         type: choice
-        default: "testnet"
+        default: 'devnet'
```

Comment:
Do we ever run this on devnet? I thought we tended to wipe devnet between releases.